cMake/FreeCAD_Helpers/PrintFinalReport.cmake

```diff
@@ -166,7 +166,7 @@ macro(PrintFinalReport)
     message(STATUS "OpenGL:              ${OPENGL_gl_LIBRARY}")
     message(STATUS "OpenGLU:             ${OPENGL_glu_LIBRARY} [${OPENGL_glu_LIBRARY}][${OPENGL_INCLUDE_DIR}]")
```

Comment:
It's weird that `${OPENGL_glu_LIBRARY}` is repeated here twice

Reply:
My guess is that it was a copy-and-paste job where the author intended to replace the first one with `OPENGL_glu_VERSION` (which doesn't exist, as far as I am aware, the OpenGL stuff doesn't have easy-to-access version info). We should probably remove it.